c/driver/postgresql/result_helper.h

```diff
@@ -89,7 +89,7 @@ class PqResultRow {
   PqResultRow() : result_(nullptr), row_num_(-1) {}
   PqResultRow(PGresult* result, int row_num) : result_(result), row_num_(row_num) {}
 
-  PqRecord operator[](const int& col_num) const {
+  PqRecord operator[](int col_num) const noexcept {
```

Comment:
Note the change in the paramater declaration from `const int&` to `int`

It may not matter, but I believe the reference type just adds an unnecessary layer of indirection

Reply:
Yeah, AFAIK there is little reason to use `const int&`